clang/test/CIR/CodeGen/vtt.cpp

```diff
@@ -5,6 +5,13 @@
 // RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fno-rtti -emit-llvm %s -o %t.ll
 // RUN: FileCheck --check-prefix=OGCG --input-file=%t.ll  %s
 
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o %t.cir
+// RUN: FileCheck --check-prefix=CIR-RTTI --input-file=%t.cir %s
```

Comment:
This is still not quite what I wanted to see. What I was hoping for were three sets of checks: CIR-COMMON, CIR-NO-RTTI, CIR-RTTI. For lines that are identical with and without RTTI (which is most of these checks), you would use the CIR-COMMON check prefix. Then the other checks would show the differences between RTTI and no RTTI.

Reply:
I will update it, thanks